ls1intum/Artemis
# `Quiz exercises`: Fix issue with drag and drop quiz exercise creation

### Checklist
#### General



#### Client

### Motivation and Context
This PR fixes the issue that is there while creating drag and drop quiz exercises.


### Steps for Testing

Prerequisites:
- 1 Instructor
- 1 Course

1. Log in to Artemis
2. Navigate to Course Administration
3. Goto Exercises of one of the courses
4. Create Drag and Drop Model Quiz by clicking on the `Create Drag and Drop Model Quiz` button followed by `Create a new Apollon Diagram`
5. Drag and drop a few elements to the canvas and click on the `Interactive ` button from the side menu.
6. Click on a few elements until it's highlighted to green then click on the `Generate a Quiz Exercise` button (As illustrated in screenshot section)
7. Observe that there are no errors, and the quiz is generated successfully.

#### Code Review
#### Manual Tests


### Screenshots
#### Before
![screen-capture (13)](https://user-images.githubusercontent.com/14681902/187088178-47e741b1-089a-4748-bea3-246ab3e8c8a8.gif)

#### After
![screen-capture (15)](https://user-images.githubusercontent.com/14681902/187088347-3a187d1d-c461-419d-83ec-f5f17fc3407f.gif)
